vars/mtaBuild.groovy

```diff
@@ -2,23 +2,18 @@ import static com.sap.piper.Prerequisites.checkScript
 
 import com.sap.piper.GenerateDocumentation
 import com.sap.piper.ConfigurationHelper
-import com.sap.piper.MtaUtils
 import com.sap.piper.Utils
+import com.sap.piper.PiperGoUtils
 import groovy.transform.Field
 
-import static com.sap.piper.Utils.downloadSettingsFromUrl
-
+@Field String METADATA_FILE = 'metadata/mtaBuild.yaml'
```

Comment:
shouldn't we keep it consistent with the other groovy steps for now?

I guess there is agreement that it should be simplified at some point in time.

Nonetheless I would not add a new way here before we have a common agreement how to go ahead here

Reply:
This is the location of file inside the resources folder of our shared lib. I don't get the point. This is basically the source from where it gets copied. What could we do different here?

@OliverNocon : I basically don't get the point here. Please explain ...

Reply:
sorry, I added the comment at the wrong place.

It was about the following three `PIPER_DEFAULTS`, `METADATA_FOLDER`, ...

What is the benefit of doing things in this step differently than in other steps already existing?

The current way of doing it is definitely not 100% nice and requires some cleanup but I don't think yet another way of doing things makes it much nicer.
I would like to strive for consistency here and then optimize it in a later step (again consistently)